### PR TITLE
remove event details button, escience to title, IS2 event specifics

### PR DESCRIPTION
**Pull request by scottyhq**
- Addresses https://github.com/uwhackweek/jupyterbook-template/issues/90 
- improves software carpentry description
- matchs style of snowex jupyter book landing page

### CODE REVIEW

File: book/preliminary/index.md [hunk 1]
```diff
@@ -9,13 +9,7 @@ We welcome all types of learners to our event, from professional software develo
...
 ## {{hackweek}} Software Carpentry Session
 
-We will be offering a preliminary training session for anyone wishing to brush up on fundamentals, or for those of you who are new to open source workflows. This will be a two day event modeled after a [Software Carpentry](https://software-carpentry.org/) training.
-
-```{note}
-**We will be hosting a {{hackweek}} Software Carpentry training as a virtual event during June 21 - 22.**
-```
-
-You can view our [Software Carpentry Schedule](swc) for more information.
+For anyone wishing to brush up on fundamentals, or for those of you who are new to open source workflows. We have a two-day recorded crash course that you can go over in advance of the hackweek: [Software Carpentry Schedule](swc).
```
Comment (JessicaS11) [inline book/preliminary/index.md:12]:
```suggestion
We strongly encourage participants to review this two-day recorded fundamentals of Python and open-source workflows crash course ([Software Carpentry Schedule](swc)) in advance of the hackweek. You may choose whichever topics you'd like to brush up on or learn. Office hours in the weeks leading up the hackweek will provide an opportunity for asking questions and troubleshooting any issues you've encountered.
```

**Comment by jomey** [comment]
Saw this a little late and #11 also removes the 'More Info" button. Has some additional user experience improvements and I can update the PR once this is merged.

File: book/application.md [hunk 2]
```diff
@@ -1,21 +1,18 @@
 # Application
 
-We are glad you are interested in applying to participate in {{hackweek}}! 
+```{warning} Application now closed
+Thank you to all people that have applied for this year's event!
+```
+
+We are glad you are interested in applying to participate in {{hackweek}}!
```
Comment (jomey) [inline book/application.md:7]: Shouldn't this be past tense now?
```
We are glad you were ...
```

**Review by jomey — APPROVED** [review]

File: book/application.md (hunk 2 above)
Comment (scottyhq) [reply book/application.md]: just removed it

**Pull request merged**